src/Dstarlite.js

```diff
@@ -76,7 +75,8 @@ export default class Dstarlite {
                 for (const {node} of predeccesors) {
                     this.updateVertex(node);
                 }
-            } 
+            }
+            loopCounter++; 
```

Comment:
smth of a nit: i think i'd prefer to do this immediately after the `while()` because *in case* a `continue` happens, we'd probably want that to be included in the iteration hard cap

also it means all lines involving `loopCounter` are right next to each other so that's a bonus 👍

Reply:
tru 🐇